### PR TITLE
add index for classification

### PR DESCRIPTION
**Pull request by tcitworld**
## Summary

Replaces #35459
I also renamed the migration since it will go into 28, not 27, and bumped the app version.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by st3iny — APPROVED** [review]

File: apps/dav/lib/Migration/Version1028Date20221114151721.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2022 Murena SAS <akhil.potukuchi.ext@murena.com>
+ *
+ * @author Murena SAS <akhil.potukuchi.ext@murena.com>
```
Comment (miaulalala) [inline apps/dav/lib/Migration/Version1028Date20221114151721.php:8]: Is this correct?

Comment (tcitworld) [reply apps/dav/lib/Migration/Version1028Date20221114151721.php:8]: Yes, @akhil1508 authored the original commit

**Review by miaulalala — APPROVED** [review]

**Comment by tcitworld** [comment]
Rebased

**Comment by miaulalala** [comment]
Failing test is due to https://github.com/nextcloud/spreed/pull/10814

Needs another rebase on master

**Pull request merged**